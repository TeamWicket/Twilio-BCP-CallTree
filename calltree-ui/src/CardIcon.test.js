import CardIcon from './CardIcon';
import React from 'react';
import { shallow } from 'enzyme';
describe("Card Icon", () => {
    it("should match snapshot of CardIcon", () => {
        expect(shallow(<CardIcon />)).toMatchSnapshot();
    })
})
