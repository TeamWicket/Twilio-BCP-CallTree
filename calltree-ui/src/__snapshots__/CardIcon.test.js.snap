// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Icon should match snapshot of CardIcon 1`] = `ShallowWrapper {}`;
